APPENGINE.md

```diff
@@ -1,4 +1,4 @@
-Use Google Cloud Java Client with Google App Engine
+Use Google Cloud Java Client with Google App Engine Java 8
```

Comment:
I'd prefer not to have the title mention Java 8; that implies that Java 7 instructions are somewhere else. We should probably have the content below state that Java 7 is not supported.

Reply:
Can we remove the "Java 8" from the title here?

Reply:
I was trying to re-enforce this is the way.  I'm not a fan of subtle messages when we are getting issues about this.

Reply:
Actually I just noticed that the title and the 2nd note below conflict... the title says these are instructions for Java 8, and the note says that the instructions aren't required for Java 8.